.github/workflows/build-test.yml

```diff
@@ -130,18 +130,47 @@ jobs:
                 TARGET_DOCKER_TEST_IMAGE: ${{ matrix.image }}
               run: ./ci/test.sh
 
-    test-mac-optional:
+    test-mac-azure-optional:
         needs: test-mac-required
         name: Tests on Mac
```

Comment:
name should be changed (or parameters applied as in my first comment) - the same also in other changed builds